### PR TITLE
June / July Updates

### PR DESCRIPTION
**Pull request by jakep**
**Includes**
- Updated totals
- Added grantee history for ACE
- Fixed link targets (videos and websites on grantee page open in new tab)
- Added this month's focus

**Open Tasks**
- [ ] Blog post link for homepage focus

### CODE REVIEW

**Comment by jakep** [comment]
@nuprin Take a peak. Looks like we just need a blog post to link to for this month's focus. Otherwise, I can add logic to hide the link when there isn't one provided. Thanks!

File: _config.yml [hunk 1]
```diff
@@ -24,36 +24,47 @@
 # twitter_username: jekyllrb
 # github_username:  jekyll
 
-contributions_to_date: $30,948
+contributions_to_date: $36,212
 
 current_focus:
-  title: Environmental Justice
+  title: Freedom to Vote
```
Comment (nuprin) [inline _config.yml]: Let's call it "Voting Rights" (the more common way to describe this issue)

File: _config.yml [hunk 2]
```diff
@@ -24,36 +24,47 @@
...
   # Note: have to cap off the body with "…" manually:
-  body: "With the United States announcing its withdrawal from the Paris Climate Agreement, our community has decided to again focus on environmental justice. Over 50% of our members voted for this focus, demonstrating how critical we feel this issue is in the fight for a more equitable and just world."
-  url: https://blog.vocalminority.us/vocal-minority-funds-climate-education-and-action-41b473dc2e42
+  body: "We're focusing this month on freedom—one of the bedrocks of our country. Specifically, our community has decided to focus on our freedom to vote. We believe that all Americans have the right to express their beliefs through the power of our voting system. Given the nationwide discussion regarding voter suppression, we've decided to fight back this month."
```
Comment (nuprin) [inline _config.yml]: Can we get it closer to something like the original prompt:

> The Presidential Advisory Commission on Election Integrity has requested highly-sensitive personal information on every U.S. voter—from social security numbers to dates of birth—in the name of preventing voter fraud. Why is the federal government marshaling considerable taxpayer resources to collect data it shouldn’t have, to reach a conclusion it cannot possibly reach, to prove something we already know isn’t true? To counter this careless act, we'll support an organization fighting to preserve the power of your vote.

It dives into the meat of the issue a little faster.

File: grantees.html [hunk 3]
```diff
@@ -21,9 +21,9 @@ <h2 class="vm-grantees__card-name">{{ grantee.name }}</h2>
             <div class="vm-grantees__card-description">{{ grantee.description }}</div>
             <div class="vm-grantees__card-links">
               {% if grantee.video %}
-                <a class="vm-grantees__card-video" href="{{ grantee.video }}"><span class="vm-grantees__card-video-button">▶︎</span> <span class="vm-grantees__card-video-inner">Watch Video</span></a>
+                <a class="vm-grantees__card-video" href="{{ grantee.video }}" target="blank"><span class="vm-grantees__card-video-button">▶︎</span> <span class="vm-grantees__card-video-inner">Watch Video</span></a>
               {% endif %}
-              <a class="vm-grantees__card-website" href="{{ grantee.website }}">View Website</a>
+              <a class="vm-grantees__card-website" href="{{ grantee.website }}" target="blank">View Website</a>
```
Comment (nuprin) [inline grantees.html]: `_blank`

File: grantees.html [hunk 4]
```diff
@@ -21,9 +21,9 @@ <h2 class="vm-grantees__card-name">{{ grantee.name }}</h2>
             <div class="vm-grantees__card-description">{{ grantee.description }}</div>
             <div class="vm-grantees__card-links">
               {% if grantee.video %}
-                <a class="vm-grantees__card-video" href="{{ grantee.video }}"><span class="vm-grantees__card-video-button">▶︎</span> <span class="vm-grantees__card-video-inner">Watch Video</span></a>
+                <a class="vm-grantees__card-video" href="{{ grantee.video }}" target="blank"><span class="vm-grantees__card-video-button">▶︎</span> <span class="vm-grantees__card-video-inner">Watch Video</span></a>
```
Comment (nuprin) [inline grantees.html]: `_blank`

**Comment by nuprin** [comment]
@jakep Here's the blog post: https://blog.vocalminority.us/our-july-focus-voting-rights-6399d98c51f0

**Comment by jakep** [comment]
@nuprin Updates pushed!

**Pull request merged**